src/elarm_alarmlist.erl

```diff
@@ -131,7 +133,7 @@ get_alarm(AlarmId, Src,
 -spec get_alarms(#al_state{}) ->
```

Comment:
The spec in line 133 uses a record, please define a type for the record and use that instead.